### ISSUE TITLE
Typescript error on deriving from PreconfiguredReconnectingEventSource

### ISSUE THREAD

**Issue by stephanpelikan** [fanout/reconnecting-eventsource#67]
Hi,

I'm using [react-sse-hooks](https://github.com/NepeinAV/react-sse-hooks) which works fine. I wanted to incorporate `ReconnectingEventSource` which has to be done as shown in [step 1](https://github.com/NepeinAV/react-sse-hooks#step-1):

```tsx
class PreconfiguredReconnectingEventSource extends ReconnectingEventSource {
  constructor(url: string | URL, eventSourceInitDict?: EventSourceInit) {
    super(url, {
      ...eventSourceInitDict,
      max_retry_time: 3000
    });
  }
};

ReactDOM.render(
  <React.StrictMode>
    <AppContextProvider>
      <EventSourceProvider eventSource={PreconfiguredReconnectingEventSource}>
        <App />
      </EventSourceProvider>
    </AppContextProvider>
  </React.StrictMode>,
  document.getElementById('root')
);
```

As you can see, I created a new class deriving from ReconnectingEventSource and passing the max_retry_time.

Unfortunately, Typescript hightlights the `eventSource` attribute of `EventSourceProvider` (defined as `eventSource?: typeof EventSource`) with this error:

**Type 'typeof PreconfiguredReconnectingEventSource' is missing the following properties from type '{ new (url: string | URL, eventSourceInitDict?: EventSourceInit): EventSource; prototype: EventSource; readonly CLOSED: number; readonly CONNECTING: number; readonly OPEN: number; }': CLOSED, CONNECTING, OPEN**

CLOSED, CONNECTING, OPEN are defined in ReconnectingEventSource but it seems to not being recognized. So, how can I get rid of that error?

Thanks,
Stephan

**Comment by harmony7** [comment]
Hi, thanks for your patience.

Today I have released 1.6.2 that should fix this.  Please give it a go.

**Comment by stephanpelikan** [comment]
Awesome! There error is gone! 🥇
